tests/launch-fastify.test.js

```diff
@@ -150,45 +151,44 @@ test('Log level inheriting system', async assert => {
     stream,
   })
   assert.strictSame(fastifyInstance.log.level, 'info')
-  const test = stream.read()
-  assert.ok(test)
+  stream.once('data', line => {
+    assert.ok(line)
+  })
+
```

Comment:
I don't understand this test. Why are there 2 fastify instance here?
I think this test can be split in two ones

Reply:
It's testing that a module can force a `logLevel` via its `options` export, and that by default (the second instance) is `info`. I've put a stream in there to avoid writing in stdout and to check that with the standard log level something is written when fastify is launched(if they ever delete that function I want to know it and replace it with something in house).
I can split it but I don't see a strong argument for it.

Reply:
Please, split into 2tests. Here we are testing the default and the custom setting

Reply:
Splitted